presto/presto.go

```diff
@@ -626,29 +627,27 @@ type driverRows struct {
 var _ driver.Rows = &driverRows{}
 
 func (qr *driverRows) Close() error {
-	if qr.nextURI != "" {
-		hs := make(http.Header)
-		hs.Add(prestoUserHeader, qr.stmt.user)
-		req, err := qr.stmt.conn.newRequest("DELETE", qr.nextURI, nil, hs)
-		if err != nil {
-			return err
-		}
-		ctx, cancel := context.WithDeadline(
-			context.Background(),
-			time.Now().Add(DefaultCancelQueryTimeout),
-		)
-		defer cancel()
-		resp, err := qr.stmt.conn.roundTrip(ctx, req)
-		if err != nil {
-			qferr, ok := err.(*ErrQueryFailed)
-			if ok && qferr.StatusCode == http.StatusNoContent {
-				qr.nextURI = ""
-				return nil
-			}
-			return err
+	if qr.nextURI == "" {
```

Comment:
note for future PRs. The changes to this method are just refactorings. Woudl be nice to have it as a separate preparatory commit.

Reply:
Duly noted